### ISSUE TITLE
nvda is not reading translation options menu in chrome

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14602]
### Steps to reproduce:
open chrome,
open a page in another language.
leave the keyboard focus on a text.
press application key
from there with the arrow until translating page and press enter
If all goes well, a window called Completed Translation should open.
walk with tabe to the button
translator options.
press enter.
a menu has opened, walk with the arrows below and right to navigate through the menu.
### Actual behavior:
nvda is silent, does not read when navigating through the menu.
### Expected behavior:
nvda should read the open menu.
narrator read this menu.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
alfa 27620
#### Windows version:
10 19045
#### Name and version of other software in use when reproducing the issue:
chrome canare 112
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
nvda portable
log
[chrome.txt](https://github.com/nvaccess/nvda/files/10546971/chrome.txt)

**Comment by cary-rowen** [comment]
Is this an alpha issue?
I can't reproduce it using 2022.4.
